src/other/dz60/dz60.json

```diff
@@ -0,0 +1,645 @@
+{
+  "name": "DZ60",
+  "vendorId": "0x445A",
+  "productId": "0x2260",
+  "lighting": "qmk_backlight",
+  "matrix": {
+    "rows": 5,
+    "cols": 15
+  },
+  "layouts": {
+    "presets": {
+      "Default": [0,0,0,0],
+      "ISO": [0,1,3,0],
+      "HHKB": [0,0,0,2],
+      "With arrows": [0,0,3,3]
+    },
+    "labels": [
+      "Split Backspace",
+      "ISO Enter",
+      [
+        "Row 4",
+        "Default",
+        "Split right shift",
+        "Arrows",
+        "Default ISO",
+        "ISO split right shift",
+        "ISO arrows",
+        "[2U] Default",
+        "[2U] Split left shift",
+        "[2U] Split right shift",
+        "[2U] Split both shifts"
+      ],
+      [
+        "Bottom row",
+        "6.25U",
+        "7U",
+        "6.25U Arrows",
+        "6.25U Arrows B",
+        "7U Arrows",
+        "Split space",
+        "Split space arrows"
+      ]
```

Comment:
These are better, thanks. Might change them later if we come up with a better way of dealing with swiss cheese PCBs ;-)